services/web/server/src/simcore_service_webserver/projects/_nodes_handlers.py

```diff
@@ -235,8 +237,37 @@ async def retrieve_node(request: web.Request) -> web.Response:
     )
 
 
+@routes.patch(
+    f"/{VTAG}/projects/{{project_id}}/nodes/{{node_id}}/outputs",
+    name="update_node_outputs",
+)
+@login_required
+@permission_required(
+    "project.node.create"
+)  # TODO use correct permission, or add a new one
+@_handle_project_nodes_exceptions
+async def update_node_outputs(request: web.Request) -> web.Response:
+    req_ctx = RequestContext.parse_obj(request)
+    path_params = parse_request_path_parameters_as(NodePathParams, request)
+    node_outputs = await parse_request_body_as(NodeOutputs, request)
+
+    ui_changed_keys = set()
+    ui_changed_keys.add(path_params.node_id)
+    return await nodes_utils.update_node_outputs(
+        app=request.app,
+        user_id=req_ctx.user_id,
+        project_uuid=path_params.project_id,
+        node_uuid=path_params.node_id,
+        outputs=node_outputs.outputs,
+        run_hash=None,
+        node_errors=None,
+        ui_changed_keys=ui_changed_keys,
```

Comment:
why not simply 
```python
return await nodes_utils.update_node_outputs(
        app=request.app,
        user_id=req_ctx.user_id,
        project_uuid=path_params.project_id,
        node_uuid=path_params.node_id,
        outputs=node_outputs.outputs,
        run_hash=None,
        node_errors=None,
        ui_changed_keys= { path_params.node_id },
)
```

Reply:
That's just my coding style. I prefer to explicitly name things to make things more readable. But I'm ok with either.